features/delete_message.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Delete Message from inbox
+    As a user
+    In order to remove messages from my inbox
+    I want be able to delete and move my message to the trash bin
+
+Background:
+  Given following users exists
+    | name   | email             | password | password_confirmation |
+    | Jenny  | jenny@ranom.com   | password | password              |
+    | Daniel | daniel@random.com | password | password              |
+
+Scenario: Deleting a message
+  Given I am logged in as "Daniel"
+  And I am on the inbox page
+  And I send a mail to "Jenny"
+  And I click "Logout" 
+  Given I am logged in as "Jenny"
```

Comment:
The steps above this one can and should be done in the background using a table
```
Given the following email exists
| sender | receiver | body          | subject           |
| Daniel  | Jenny    | some message | some subject  |
```

Reply:
done